target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -86,6 +86,10 @@ netgear,ex6150v2)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:router" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:client" "phy1tpt"
 	;;
+teltonika,rutx11)
```

Comment:
Can this be handled directly in the `.dts` file (as it is for teltonika,rutx10)?

Reply:
Fixed, testing.